pkg/services/authz/zanzana/server/authz_server.go

```diff
@@ -56,6 +57,18 @@ func WithSchema(modules []transformer.ModuleFile) ServerOption {
 	}
 }
 
+func NewAuthzServer(cfg *setting.Cfg, openfga openfgav1.OpenFGAServiceServer) (*Server, error) {
+	stackID := cfg.StackID
+	if stackID == "" {
+		stackID = "default"
+	}
+
+	return NewAuthz(
+		openfga,
+		WithTenantID(fmt.Sprintf("stack-%s", stackID)),
```

Comment:
If this has to be aligned with the namespace:
```suggestion
		WithTenantID(fmt.Sprintf("stacks-%s", stackID)),
```
Otherwise ignore this :)